File: Java/10809-알파벳 찾기.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] arr = new int[26];
        
        for(int i = 0; i < 26; i++){
            arr[i] = -1;
        }
        
        String word = sc.next();
        
        for(int i = 0; i < word.length(); i++){
            if(arr[word.charAt(i)-'a'] == -1){
                arr[word.charAt(i)-'a'] = i;
            }  
        }
        
        for(int i = 0; i < 26; i++){
            System.out.printf("%d ", arr[i]);
        }
    }
}

File: C++/10809-알파벳 찾기.cpp
#include <iostream>
#include <cstring>
using namespace std;

int main() {
    int i, arr[26];
    string word;
    
    for(i = 0; i < 26; i++){
        arr[i] = -1;
    }
    
    cin>>word;
    
    for(i = 0; i < word.size(); i++){
        if(arr[word[i]-'a'] == -1){
            arr[word[i]-'a'] = i;
        }  
    }
    
    for(i = 0; i < 26; i++){
        cout<<arr[i]<<' ';
    }
    
    return 0;
}
